clang/test/CheckedC/headers/nostdlibinc.c

```diff
@@ -0,0 +1,78 @@
+// The -nostdlibinc option suppresses the search for include files in standard
+// system include directories like /usr/local/include, /usr/include,
+// /usr/include/<TARGET_ARCH>. But the <BUILD>/lib/clang/<VERSION>/include
+// directory is still searched for include files.
+// Therefore, in the context of a Checked C compilation, all the wrapper
+// header files listed below are found while the corresponding system header
+// file that is included in each of these files (via #include_next) is not
+// found when -nostdlibinc is specified on the compilation command line.
+//
+// This test confirms the above behavior with the following checks:
+//   1) The wrapper header files listed below are found.
+//   2) The system header file of the same name that each wrapper header file
+//      includes (via #include_next), is not found.
+// The -MM -MG preprocessor options list the header files not found.
+//
+// RUN: %clang -target x86_64-unknown-unknown -nostdlibinc -ffreestanding -MM -MG %s | FileCheck -check-prefix=CHECK %s
```

Comment:
By default, `FileCheck` will always check the prefix `CHECK`. So you can omit the `--check-prefix=CHECK` part in this `RUN` command.
See https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-filecheck-check-prefix

Reply:
Done.